cmd/tk/workflow.go

```diff
@@ -37,21 +37,35 @@ func applyCmd() *cobra.Command {
 	vars := workflowFlags(cmd.Flags())
 	force := cmd.Flags().Bool("force", false, "force applying (kubectl apply --force)")
 	autoApprove := cmd.Flags().Bool("dangerous-auto-approve", false, "skip interactive approval. Only for automation!")
+	willDelete := cmd.Flags().Bool("enable-delete", false, "enable deletion of removed resources")
 	cmd.Run = func(cmd *cobra.Command, args []string) {
 		if kube == nil {
 			log.Fatalln(kubernetes.ErrorMissingConfig{Verb: "apply"})
 		}
 
-		raw, err := evalDict(args[0])
+		environment := args[0]
+
+		raw, err := evalDict(environment)
 		if err != nil {
 			log.Fatalln("Evaluating jsonnet:", err)
 		}
 
-		desired, err := kube.Reconcile(raw, stringsToRegexps(vars.targets)...)
+		desired, err := kube.Reconcile(raw, environment, stringsToRegexps(vars.targets)...)
 		if err != nil {
 			log.Fatalln("Reconciling:", err)
 		}
 
+		if *willDelete {
+			deletions, err := kube.ReconcileDeletions(desired, kubernetes.ReconcileDeletionsOpts{Environment: environment})
+			if err != nil {
+				log.Fatalln("Reconciling deletions:", err)
+			}
+			err = kube.DeleteResources(deletions)
```

Comment:
you should pass `kubernetes.ApplyOpts{Force: *force, AutoApprove: *autoApprove}` to kube.DeleteResources